ci-operator/config/openshift-priv/cluster-control-plane-machine-set-operator/openshift-priv-cluster-control-plane-machine-set-operator-release-4.16.yaml

```diff
@@ -176,6 +176,17 @@ tests:
         requests:
           cpu: 100m
     workflow: openshift-e2e-vsphere-zones
+- as: vsphere-ipi-zones-f28-longduration-cloud
```

Comment:
Does this test trigger when https://github.com/openshift/cluster-control-plane-machine-set-operator repo changes? We haven't CPMS long duration cases for vSphere in our automation at present.

Reply:
Yes that is the plan , we can replace everything to may be just runs , what CPMS cases we have , this is to make sure we have the reference PR 
so for example we can just run - 53081 by modifying name and other details like below at [openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml]
```
`- as: vsphere-ipi-zones-f28-longduration-cloud
  .
  .
  steps:
    cluster_profile: vsphere-multizone-2
    env:
      TEST_SCENARIOS: 53081
      TEST_TIMEOUT: "90"
    test:
    - ref: openshift-test-private
    workflow: cucushift-installer-rehearse-vsphere-ipi-zones
 `
```
but before any such changes want to make sure , if this is the approach we are good to go ahead with. Later on we might include `CPMS` in general like here mentioned `longduration`